spec/bolt/catalog_spec.rb

```diff
@@ -59,21 +60,85 @@
           'target_vars' => {},
           'target_facts' => {},
           'target_features' => {}
-        },
-        'config' => {
-          'transport' => "ssh",
-          'transports' => {
-            'ssh' => { 'connect-timeout' => 10, tty: false, "host-key-check" => true },
-            'winrm' => { 'connect-timeout' => 10, tty: false, ssl: true, "ssl-verify" => true },
-            'pcp' => { 'connect-timeout' => 10,
-                       'tty' => false,
-                       'task-environment' => 'production' },
-            'local' => { 'connect-timeout' => 10, tty: false }
-          }
+        }
```

Comment:
Not to drag this out, but while we're updating this 'inventory' can be removed entirely from this, it looks like it's not used anywhere in `catalog.rb` or the test itself.